proj/src/actions/statisticsActions.js

```diff
@@ -0,0 +1,23 @@
+import * as AT from 'actions/types'
+
+export const fetchData = () => ({
```

Comment:
on the same principle fetchData is to generic. data of what? statistics. so maybe fetchStatistics would be more appropriate